cmd/run.go

```diff
@@ -100,5 +96,39 @@ func NewRunCommand(run *KoolRun) (runCmd *cobra.Command) {
 
 	// after a non-flag arg, stop parsing flags
 	runCmd.Flags().SetInterspersed(false)
+
+	// look for kool.yml on current working directory
+	_ = run.parser.AddLookupPath(run.envStorage.Get("PWD"))
+	// look for kool.yml on kool folder within user home directory
+	_ = run.parser.AddLookupPath(path.Join(run.envStorage.Get("HOME"), "kool"))
+
+	if usageTempl, err := getRunUsageTemplate(run, runCmd); err == nil {
```

Comment:
at first I am a bit concerned of the overhead we add to the `init()` doing all the parse in order to bootstrap the commands only... but I guess UX wise it is worth. Nice :+1:

Reply:
Now, about ignoring the error... maybe in case we have `KOOL_VERBOSE=1` we should then print this omitted error anyway?

Reply:
@danielsuguimoto we can look into it/evaluate in a next iteration!